src/pyggdrasil/tree_inference/_config.py

```diff
@@ -0,0 +1,63 @@
+"""Config classes for MCMC sampler,
+ cell simulation and tree inference, distance calculation."""
+
+from pydantic import BaseModel, validator
+
+
+class MoveProbConfig(BaseModel):
+    """Move probabilities for MCMC sampler."""
+
+    prune_and_reattach: float = 0.1
+    swap_node_labels: float = 0.65
+    swap_subtrees: float = 0.25
+
+    @validator("prune_and_reattach", "swap_node_labels", "swap_subtrees")
+    def move_prob_validator(cls, v):
```

Comment:
Nice!

Reply:
Oh, yes -  implemented with the above `confloat` and precision.